config/jobs/kubernetes/sig-scalability/sig-scalability-golang.yaml

```diff
@@ -55,7 +55,7 @@ periodics:
 
 - interval: 4h
   name: ci-golang-tip-k8s-1-23
-  cluster: scalability
+  cluster: k8s-infra-prow-build
```

Comment:
We can also change project to --gcp-project-type=scalability-project I think

Reply:
Right, good catch, thanks!